qutebrowser/misc/sql.py

```diff
@@ -17,10 +17,11 @@
 # You should have received a copy of the GNU General Public License
 # along with qutebrowser.  If not, see <https://www.gnu.org/licenses/>.
 
-"""Provides access to an in-memory sqlite database."""
+"""Provides access to an in-memory sqlite database."""  # I guess this is an old comment
```

Comment:
Huh, indeed, that's from back when the history was stored in a text file and just loaded into sqlite at start.

What about something like:

```suggestion
"""Provides access to sqlite databases."""
```